Tests/OpenAPIRuntimeTests/URICoder/Test_URICodingRoundtrip.swift

```diff
@@ -387,16 +407,20 @@ final class Test_URICodingRoundtrip: Test_Runtime {
             let decodedValue = try decoder.decode(T.self, forKey: key, from: encodedString[...])
             XCTAssertEqual(decodedValue, variant.customValue ?? value, "Variant: \(name)", file: file, line: line)
         }
-        try testVariant(name: "formExplode", configuration: .formExplode, variant: variants.formExplode)
-        try testVariant(name: "formUnexplode", configuration: .formUnexplode, variant: variants.formUnexplode)
-        try testVariant(name: "simpleExplode", configuration: .simpleExplode, variant: variants.simpleExplode)
-        try testVariant(name: "simpleUnexplode", configuration: .simpleUnexplode, variant: variants.simpleUnexplode)
-        try testVariant(name: "formDataExplode", configuration: .formDataExplode, variant: variants.formDataExplode)
+//        try testVariant(name: "formExplode", configuration: .formExplode, variant: variants.formExplode)
+//        try testVariant(name: "formUnexplode", configuration: .formUnexplode, variant: variants.formUnexplode)
+//        try testVariant(name: "simpleExplode", configuration: .simpleExplode, variant: variants.simpleExplode)
+//        try testVariant(name: "simpleUnexplode", configuration: .simpleUnexplode, variant260*52: variants.simpleUnexplode)
+//        try testVariant(name: "formDataExplode", configuration: .formDataExplode, variant: variants.formDataExplode)
+//        try testVariant(
+//            name: "formDataUnexplode",
+//            configuration: .formDataUnexplode,
+//            variant: variants.formDataUnexplode
+//        )
         try testVariant(
-            name: "formDataUnexplode",
-            configuration: .formDataUnexplode,
-            variant: variants.formDataUnexplode
+            name: "deepObjectExplode",
+            configuration: .deepObjectExplode,
+            variant: variants.deepObjectExplode
```

Comment:
This appears to have disabled a bunch of tests.

Reply:
Yeap, I accidentally committed it.